starfyre/__init__.py

```diff
@@ -8,15 +8,15 @@
 from .parser import RootParser
 
 
-def create_component(pyml="", css="", js="", client_side_python=""):
+def create_component(pyml="", css="", js="", client_side_python="", class_name=""):
```

Comment:
`class_name` is used to track what class had the issue.

Reply:
Can you please explain a bit more in regards to why is it needed?

Reply:
> This class_name will be used to track that they have an issue at line x in class_name. I believe it will help the users quickly spot the file that is causing the issue - considering a big project.


Ah, I see. Let us come up with a different name for this variable. `class_name` means something else in the context of frontend. See here - https://legacy.reactjs.org/docs/faq-styling.html

Reply:
Thanks. I will change the name